manifests/job.pp

```diff
@@ -28,23 +28,17 @@
 #     Provide a command to execute to compare Jenkins job files
 #
 define jenkins::job(
-  $config,
-  $source   = undef,
-  $template = undef,
-  $jobname  = $title,
-  $enabled  = undef,
-  $ensure   = 'present',
-  $difftool = '/usr/bin/diff -b -q',
+  String $config,
+  Optional[Stdlib::Absolutepath] $source    = undef,
+  Optional[Stdlib::Absolutepath] $template  = undef,
+  String $jobname                           = $title,
+  $enabled                                  = undef,
```

Comment:
It might be more useful to add `Any` here. Now it's weird that there is nothing.